products/kotlin.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Kotlin
+layout: post
+category: lang
+iconSlug: kotlin
+permalink: /kotlin
+alternate_urls:
+  - /kotlinlang
+command: kotlinc-native -version
+link: https://kotlinlang.org/docs/releases.html
+sortReleasesBy: "cycleShortHand"
+changelogTemplate: "https://github.com/JetBrains/kotlin/releases/tag/v__LATEST__"
+
+activeSupportColumn: false
+discontinuedColumn: false
+releaseDateColumn: true
+releaseColumn: true
+
+releases:
+  - releaseCycle: "1.6"
+    cycleShortHand: 106
+    eol: false #not sure about eol dates of kotlin if you find any information about this please change this part
+    support: false
+    release: 2021-11-16
+    latest: "1.6.0"
+    link: https://blog.jetbrains.com/kotlin/2021/11/kotlin-1-6-0-is-released/
+
+  - releaseCycle: "1.5"
+    cycleShortHand: 105
+    eol: false #not sure about eol dates of kotlin if you find any information about this please change this part
+    support: false
+    release: 2021-05-05
+    latest: "1.5.31"
+
+  - releaseCycle: "1.4"
+    cycleShortHand: 105
```

Comment:
Sorting is wrong due duplicate `cycleShortHand: 105`. I think the 1.4 should have `cycleShortHand: 104`